writer.go

```diff
@@ -35,11 +35,26 @@ type Writer struct {
 	BypassValidation bool
 }
 
+// WriteOpts defines options for writing a file.
+type WriteOpts struct {
+	// LineEnding sets a custom line ending character.
+	LineEnding string `json:"lineEnding"`
+}
+
 // NewWriter returns a new Writer that writes to w.
 func NewWriter(w io.Writer) *Writer {
+	return NewWriterWithOpts(w, nil)
+}
+
+// NewWriter returns a new Writer that writes to w.
+func NewWriterWithOpts(w io.Writer, opts *WriteOpts) *Writer {
+	lineEnding := "\n"
```

Comment:
We could try to infer the default from the running operating system here, e.g. look at `runtime.GOOS`, if desired.

Reply:
We've avoided that because it's hard to know what an HTTP server should return. If the server and client are on different OS's then who wins?